docs/decisions/006-semantic-class-names-for-themes.md

```diff
@@ -0,0 +1,176 @@
+| background            | background-foreground    |                                       |
+| card                  | card-foreground          |                                       |
+| popover               | popover-foreground       |                                       |
+| primary               | primary-foreground       |                                       |
+| secondary             | secondary-foreground     |                                       |
+| muted                 | muted-foreground         |                                       |
+| accent                | accent-foreground        |                                       |
+| destructive           | destructive-foreground   |                                       |
+| border                | -                        |                                       |
+| input                 | -                        |                                       |
+| ring                  | -                        |                                       |
+| radius                | -                        | Not a color, but included for completeness |
+| header                | header-foreground        | **Custom**                            |
+| footer                | footer-foreground        | **Custom**                            |
+| callout               | callout-foreground       | **Custom**                            |
+| brand-50 ... brand-950| brand-foreground-50 ... brand-foreground-950 | **Custom** Brand color scale         |
+| success               | -                        | **Custom** Status color               |
+| warning               | -                        | **Custom** Status color               |
+| error                 | -                        | **Custom** Status color               |
+
+We are extending the Shadcn set with custom variables for `header`, `footer`, `callout`, and a full `brand` color scale (with matching foregrounds). We are also adding `success`, `warning`, and `error` for status messaging. Some Shadcn variables like `sidebar` and chart colors are omitted for now, as we don't expect to use them.
+
+The `brand` color scale is intended to be a bail out for designers when we need another color that the other semantic class names don't provide. An idea is to let a content editor choose a section's (i.e. block) background color from a dropdown of a subset of tailwind colors (associated with classnames) and their brand-50 to brand-950 colors. But they may be used elsewhere too.
+
+This approach gives us a consistent set of semantic class names for theming, while allowing flexibility for our own design needs.
+
+### Toggling themes
+
+We will use this CSS Variables pattern to toggle themes between avalanche centers. Each avalanche center could have a light mode and a dark mode.
+
+We would have a theme defined in our globals.css for each avalanche center like:
+
+```
+// globals.css
```

Comment:
Are we not tackling colors chosen by admins in Payload out of principle, or to make our progress today expedient?

Reply:
Oops sorry, didn't read all the way to the bottom, disregard.